modules/cms-kit/test/Volo.CmsKit.Application.Tests/Pages/PageAdminAppService_Tests.cs

```diff
@@ -38,39 +66,41 @@ public async Task ShouldCreateAsync()
             var dto = new CreatePageInputDto
             {
                 Title = "test",
-                Url = "test-url"
+                Url = "test-url",
+                Content = "test-content"
             };
 
-            await Should.NotThrowAsync(async () => await _pageAdminAppService.CreatePageAsync(dto));
+            await Should.NotThrowAsync(async () => await _pageAdminAppService.CreateAsync(dto));
 
             var page = await _pageRepository.GetByUrlAsync(dto.Url);
             
             page.Title.ShouldBe(dto.Title);
         }
         
         [Fact]
-        public async Task ShouldNotCreateAsync()
+        public async Task ShouldNotCreateExistUrlAsync()
         {
             var dto = new CreatePageInputDto
             {
                 Title = "test",
-                Url = _data.Page_1_Url
+                Url = _data.Page_1_Url,
+                Content = "test-content"
             };
 
-            await Should.ThrowAsync<Exception>(async () => await _pageAdminAppService.CreatePageAsync(dto));
+            await Should.ThrowAsync<Exception>(async () => await _pageAdminAppService.CreateAsync(dto));
```

Comment:
We should create a spesific business exception here. What kind of exception does this call throw?